tests/test_examples.py

```diff
@@ -38,6 +40,8 @@ def run_tests():
 def main():
     # Create tests for each example
     for _, name, _ in pkgutil.iter_modules(['examples']):
+        if name in examples_skipped:
+            continue
         test_func = make_test_function(name)
         setattr(TestExamples, 'test_{0}'.format(name), test_func)   
```

Comment:
```suggestion
        if name not in examples_skipped:
            test_func = make_test_function(name)
            setattr(TestExamples, 'test_{0}'.format(name), test_func)   
```

Accidentally committed the change instead of suggesting it earlier. I ran simpler code under this control flow as well as my suggestion and saw some performance improvement in this change. Might be worth using for slightly increased efficiency?

Reply:
Good catch!